csrankings-g.csv

```diff
@@ -496,7 +496,7 @@ Giulio Jacucci,University of Helsinki,https://researchportal.helsinki.fi/en/pers
 Giulio Malavolta [SP],Max Planck Society,https://sites.google.com/view/giuliomalavolta,_pYEarUAAAAJ
 Giuseppe A. Di Lucca,University of Sannio,https://www.ding.unisannio.it/persone/docenti/di-lucca,pHMonFwAAAAJ
 Giuseppe Antonio Di Lucca,University of Sannio,https://www.ding.unisannio.it/persone/docenti/di-lucca,pHMonFwAAAAJ
-Giuseppe Ateniese,Stevens Institute of Technology,http://web.stevens.edu/facultyprofile/?id=2182,EyZJ08MAAAAJ
+Giuseppe Ateniese,George Mason University,https://ateniese.github.io,EyZJ08MAAAAJ

```

Comment:
Web page still says Stevens. Please point to something indicating new affiliation.